lms/envs/aws.py

```diff
@@ -156,6 +156,7 @@
 SESSION_COOKIE_DOMAIN = ENV_TOKENS.get('SESSION_COOKIE_DOMAIN')
 SESSION_COOKIE_HTTPONLY = ENV_TOKENS.get('SESSION_COOKIE_HTTPONLY', True)
 SESSION_COOKIE_SECURE = ENV_TOKENS.get('SESSION_COOKIE_SECURE', SESSION_COOKIE_SECURE)
+SESSION_COOKIE_AGE = ENV_TOKENS.get('SESSION_COOKIE_AGE', 86400)
```

Comment:
We need to move this setting to `lms/envs/common.py` so that this settings available for all environments.
```python
SESSION_COOKIE_AGE = ENV_TOKENS.get('SESSION_COOKIE_AGE', 86400)  # Default: 24 hours, in seconds
```